psycopg/psycopg/connection.py

```diff
@@ -265,29 +265,36 @@ def rollback(self) -> None:
     def cancel_safe(self, *, timeout: float = 30.0) -> None:
         """Cancel the current operation on the connection.
 
+        :param timeout: raise a `~errors.CancellationTimeout` if sending
+            the timeout doesn't succeed within `timeout` seconds.
```

Comment:
I suppose you meant: "if sending the *cancellation* doesn't succeed". But in fact, it's not clear to me why you're talking about "sending the cancellation". How about: "if the cancellation request does not succeed within `timeout` seconds." ?